lldb/source/Expression/DWARFExpression.cpp

```diff
@@ -1657,7 +1650,7 @@ llvm::Expected<Value> DWARFExpression::Evaluate(
     case DW_OP_eq:
       if (stack.size() < 2) {
         return llvm::createStringError(
-            "Expression stack needs at least 2 items for DW_OP_eq.");
+            "expression stack needs at least 2 items for DW_OP_eq");
```

Comment:
The obvious next step to refactor will be to hoist the arity check of the operators into a single helper that uses a table and a %s in the error message :-)

Reply:
Sounds like a job for Dwarf.def!

Reply:
https://github.com/llvm/llvm-project/pull/94679